packages/wasm/src/build.ts

```diff
@@ -28,38 +37,29 @@ export const witness = (txPlan: TransactionPlan, sct: StateCommitmentTree): Witn
   return WitnessData.fromJsonString(JSON.stringify(result));
 };
 
-const githubSourceDir =
-  'https://github.com/penumbra-zone/penumbra/raw/main/crates/crypto/proof-params/src/gen/';
-
-const provingKeys = [
-  { keyType: 'spend', file: 'spend_pk.bin' },
-  { keyType: 'output', file: 'output_pk.bin' },
-  { keyType: 'swap', file: 'swap_pk.bin' },
-  { keyType: 'swap_claim', file: 'swapclaim_pk.bin' },
-  { keyType: 'nullifier_derivation', file: 'nullifier_derivation_pk.bin' },
-  { keyType: 'delegator_vote', file: 'delegator_vote_pk.bin' },
-  { keyType: 'undelegate_claim', file: 'undelegateclaim_pk.bin' },
-];
-
-const loadProvingKeys = async () => {
-  const promises = provingKeys.map(async ({ file, keyType }) => {
-    const response = await fetch(`${githubSourceDir}${file}`);
-    if (!response.ok) throw new Error(`Failed to fetch ${file}`);
-
-    const buffer = await response.arrayBuffer();
-    wasmLoadProvingKey(buffer, keyType);
-  });
```

Comment:
I'm personally more of a fan of having this array and just looping over it like so. It really only looks like you need to do:

```diff
- const response = await fetch(`${githubSourceDir}${file}`);
+ const response = await fetchBinaryFile(`${githubSourceDir}${file}`);
```

Reply:
I added a `utils.ts` file in the `wasm` package with this pattern.